test/run/t902.py.real

```diff
@@ -1,7 +1,7 @@
-TypeError: first argument must be callable on line 4
-TypeError: first argument must be callable on line 9
-TypeError: object is not iterable on line 14
-KeyError: 5 on line 20
+TypeError('first argument must be callable')
+TypeError('first argument must be callable')
+TypeError('object is not iterable')
+KeyError('5')
 None
-TypeError: __missing__() takes at most 1 arguments (2 given) on line 30
-KeyError: {1: 2} on line 35
+TypeError('__missing__() takes at most 1 arguments (2 given)')
+KeyError('{1: 2}')
```

Comment:
something like this still needs to be forced:
should be...
`__missing__() takes exactly one argument (2 given)`

Reply:
and this is a different error message to py3 🤔

Reply:
the only difference between py3 and py2 I see is:
py3: `TypeError('expected at most 1 arguments, got 2')`
py2: `TypeError('expected at most 1 arguments, got 2',)`

Reply:
we have this every so often we realise we should be running our tests agains python as well as against skulpt. 😄 we used to do that which is why we have these force files.

Reply:
> the only difference between py3 and py2 I see is:
> py3: TypeError('expected at most 1 arguments, got 2')
> py2: TypeError('expected at most 1 arguments, got 2',)

Ah yes - i meant to mention that. I chose the py3 version... and didn't see a reason to go with the `py2` version
So should I leave the force files in in that case?